addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -847,6 +847,11 @@ section, .oe_img_bg, [data-oe-shape-data] {
         -webkit-background-clip: text;
         -webkit-text-fill-color: transparent;
     }
+    // Neutralize text-gradient if another color is set within.
+    font[style*="color"]:not(.text-gradient), font[style*="background-image"]:not(.text-gradient), [class*="text-"]:not(.text-gradient) {
+        -webkit-background-clip: initial;
+        -webkit-text-fill-color: initial;
+    }
```

Comment:
I am not sure to understand how this is possible as if you try to color part of a text which uses a gradient, the gradient gets split anyway:

![image](https://user-images.githubusercontent.com/10338094/144054840-8793b6ba-5c09-41a7-a7c5-f5943bc3a493.png)

Reply:
Removed: limit this PR to style changes such as bold and italic - not nested colors because of the way the editor avoids nesting `<font>` tags to apply colors.